qcodes/instrument/sims/lakeshore_model336.yaml

```diff
@@ -0,0 +1,42 @@
+spec: "1.0"
+devices:
+  device 1:
+    eom:
+      GPIB INSTR:
+        q: "\n"
+        r: "\n"
+    error: ERROR
+    dialogues:
+      - q: "*IDN?"
+        r: "QCoDeS, m0d3l, 336, 0.0.01"
+
+    properties:
+      temperature:
+        default: 100.0
+        getter:
+          q: "KRDG? A"
+          r: "{}"
+
+      sensor_raw:
+        default: 101.0
+        getter:
+          q: "SRDG? A"
+          r: "{}"
+
+      sensor_status:
+        default: 0
+        getter:
+          q: "RDGST? A"
+          r: "{}"
+
+      sensor_name:
+        default: "my name is boring"
+        getter:
+          q: "INNAME? A"
+          r: "{}"
+        setter:
+          q: "INNAME A,\"{}\""
+          r: OK
```

Comment:
This "OK" response is now obsolete (#968).

Reply:
OK

Reply:
I dont know if there was an actual test file using this file that I missed in the rebase